haiku-apps/batchrename/batchrename-0.2.0.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="An application and Tracker add-on to rename multiple files at once"
+DESCRIPTION="Renamer is a standalone application, and Tracker add-on for Haiku. \
+You can rename multiple files in multiple directories at once, and also shows \
+a preview. It offers many renaming options, and can also rename files by contents \
+of their attributes, or the result of shell scripts in case you need that."
+HOMEPAGE="https://github.com/axeld/rename"
+COPYRIGHT="2019-2024 pinc Software"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/axeld/rename/archive/b768c8525dbaf15b055ffd0b303b8c79a17d8c9a.tar.gz"
+CHECKSUM_SHA256="d94bd208ca03f46f58ed4daef4a044810630a86df60cf1431912375116a5f229"
+SOURCE_DIR="rename-b768c8525dbaf15b055ffd0b303b8c79a17d8c9a"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+#USER_SETTINGS_FILES="settings/pinc.rename settings"
+
+PROVIDES="
+	batchrename$secondaryArchSuffix
+	cmd:rename= $portVersion
+	app:rename= $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	icu74${secondaryArchSuffix}_devel
```

Comment:
Usually, references to library names are used instead of package names, so something like `devel:libicuuc$secondaryArchSuffix >= 74`. But this is just a cosmetic change in this case.

Reply:
I actually only need the headers, but if your version work, then it would definitely be preferred, as I'm pretty agnostic about the actual version. Thanks for the suggestion!

Reply:
If the version doesn't matter, you can also leave out the version constraint then. For gcc2, there is only icu-57 anyway (the package name has no version suffix). With the `devel:` syntax, you don't need to differentiate that then.